build/Dockerfile

```diff
@@ -0,0 +1,50 @@
+ARG UBUNTU_VERSION=16.04
+
+# Use Ubuntu as the base image
+FROM ubuntu:${UBUNTU_VERSION}
```

Comment:
for Fibs how do ew make sure that this image is ok?

Reply:
We tested and merged this as part of [this PR](https://github.com/newrelic/nri-docker/pull/261)